Mathlib/LinearAlgebra/BilinearForm/Basic.lean

```diff
@@ -64,7 +64,7 @@ namespace BilinForm
 
 #noalign bilin_form.coe_fn_mk
 
-@[deprecated]
+@[deprecated] -- 2024-04-14
 theorem coeFn_congr : ∀ {x x' y y' : M}, x = x' → y = y' → B x y = B x' y'
```

Comment:
Shouldn't these also use the "since" syntax?

Reply:
Updated!